models/intel/bert-small-uncased-whole-word-masking-squad-0001/description/bert-small-uncased-whole-word-masking-squad-0001.md

```diff
@@ -0,0 +1,59 @@
+# bert-small-uncased-whole-word-masking-squad-0001
+
+## Use Case and High-Level Description
+
+This is a small BERT-large like model distilled on SQuAD v1.1 training set from original
+bert-large-uncased-whole-word-masking-finetuned-squad provided by the transformes library (https://github.com/huggingface/transformers).
+The model performs question answering for English language;
+the input is a concatenated premise and question for the premise,
+and the output is the location of the answer to the question inside the premise.
+For details about the original model, check out
+[BERT: Pre-training of Deep Bidirectional Transformers for Language Understanding](https://arxiv.org/abs/1810.04805).
+[HuggingFace's Transformers: State-of-the-art Natural Language Processing](https://arxiv.org/abs/1910.03771).
+
+Tokenization occurs using the BERT tokenizer (see the demo code for implementation details) and the enclosed `vocab.txt` dictionary file. Input is to be lower-cased before tokenizing.
+
+## Specification
+
+| Metric            | Value                 |
+|-------------------|-----------------------|
+| GOps              | 23.9                  |
+| MParams           | 57.94                 |
+| Source framework  | PyTorch\*             |
+
+## Accuracy
+
+The quality metrics were calculated on the SQuAD v1.1 dataset ("dev" split). Maximum sequence length is 384, input is lower-cased.
+
+| Metric                    | Value         |
+|---------------------------|---------------|
+| F1                        |        91.57% |
+| Exact match (EM)          |        85.04% |
+
+## Performance
+
+## Input
+
+Input `input_ids`: A `1,384` sequence of tokens (integer values) representing the tokenized premise and question (`input_ids`). The sequence structure is as follows (`[CLS]`, `[SEP]` and `[PAD]` should be replaced by corresponding token IDs as specified by the dictionary):
```

Comment:
Also, please explain what do the `inputs_ids`, `mask_ids` and `segments_ids` mean? They differ from input name, can it be a mistake?

Reply:
Sorry for mess. The name has to be the same. will fix. Thank for catch.